vignettes/gsmExtensions.Rmd

```diff
@@ -0,0 +1,422 @@
+
+
+### `/1_mappings`
+
+The mappings folder contains all of the mappings from `Raw_*` data to `Mapped_*` data. Each file within this directory is to be named for the data table it is creating, minus the `Mapped_` suffix. The `yaml`s will contain the three required sections, which are discussed in detail in the `Module Configuration` section above. The `yaml`s in this folder will be combined via `CombineSpecs()` to create a master spec that defines all necessary tables and columns for the module(s) in this package.
+
+Below are two examples of these mapping `yaml` files- the first which requires no transformations, and is very simple, and the second which requires multiple steps to produce the desired mapped data. 
+
+#### Mapped_AE mapping `yaml` file
+
+```
+meta:
+  Type: Mapped
+  ID: AE
+  Description: Adverse Event Data Mapping 
+  Priority: 1
+spec: 
+ Raw_AE:
+    subjid:
+      required: true
+      type: character
+    aeser:
+      required: true
+      type: character
+steps:
+  - output: Mapped_AE
+    name: =
+    params:
+      lhs: Mapped_AE
+      rhs: Raw_AE
+```
+#### Mapped_DATACHG mapping `yaml` file
+```
+meta:
+  Type: Mapped
+  ID: DATACHG
+  Description: Data Changes Data Mapping 
+  Priority: 2
+spec: 
+  Raw_DATACHG:
+    subject_nsv:
+      required: true
+      type: character
+      source_col: subjectname
+    n_changes:
+      required: true
+      type: numeric
+  Mapped_SUBJ:
+    subjid:
+      required: true
+      type: character
+    subject_nsv:
+      required: true
+      type: character
+steps:
+  # Merge [ subjid ] onto EDC domains.
+  - output: Temp_SubjectLookup
+    name: select
+    params:
+      .data: Mapped_SUBJ
+      subjid: subjid
+      subject_nsv: subject_nsv
+  - output: Mapped_DATACHG
+    name: left_join
+    params:
+      x: Raw_DATACHG
+      "y": Temp_SubjectLookup
+      by: subject_nsv
+```
+
+### `/2_metrics`
+
+The metrics directory contains all of the workflows that perform analysis steps, converting mapped data into metrics that are displayed in a report. In the case of `{gsm}`, these metrics are the 12 Key Risk Indicators, calculated at both the site- and country-level, that are discussed in the Data Analysis Step-by-Step Vignette. Each `yaml` in this file produces a list of analysis data tables that capture the formatted input table, the transformed table, the flagged table, and the summary table. In general, these `yaml`s should at least provide a summary table that contains statistics about the metric at the specified level of aggregation.
+
+Examples of these yamls can be found above in the `Module Configuration` section, as well as in the Data Analysis Vignette.
```

Comment:
```suggestion
Examples of these `yaml`s can be found above in the `Module Configuration` section, as well as in the (`vignette("DataAnalysis")`) vignette.
```

i don't mind the lack of ticks plural form of yamls actually, maybe thats a good option, but highlighting for consistency and link to the appropriate vignette